package.json

```diff
@@ -30,10 +30,12 @@
     "packages:clean": "lerna exec --parallel -- yarn clean",
     "build:mozilla": "echo 'Packaging for Mozilla based browsers.' && web-ext build --config ./web-ext.config.js -s ./src -a ./dist/web-ext/mozilla",
     "build:chrome": "echo 'Packaging for Chromium based browsers.' && bash ./scripts/build-chrome.sh",
-    "build:lint": "web-ext lint --config ./web-ext.config.js -s ./src"
+    "build:lint": "web-ext lint --config ./web-ext.config.js -s ./src",
+    "build:privacy-policy-html": "mkdir -p ./dist && marked -i PRIVACY.md -o ./dist/privacy-policy.html"
```

Comment:
I assume you don't really need the `-p`,  since you're creating a subfolder only one level deep?

Reply:
I always include it for convenience. :D